src/Log/Engine/BaseLog.php

```diff
@@ -95,12 +96,16 @@ protected function _format($data, array $context = [])
 
         $object = is_object($data);
 
+        if ($object && $data instanceof EntityInterface) {
+            return json_encode($data, JSON_UNESCAPED_UNICODE | JSON_PRETTY_PRINT);
+        }
+
         if ($object && method_exists($data, '__toString')) {
             return (string)$data;
         }
 
         if ($object && $data instanceof JsonSerializable) {
-            return json_encode($data);
+            return json_encode($data, JSON_UNESCAPED_UNICODE);
```

Comment:
I decided to treat `EntityInterface` separate from `JsonSerializable` because I wanted to keep the `JSON_PRETTY_PRINT` used in `EntityTrait::__toString()`. Doing so for `JsonSerializable` as well would have changed even more of the current logging (including breaking `FileLogTest`).